data/raw/keybindings.json

```
@@ -2896,7 +2896,7 @@
   },
   {
     "type": "keybinding",
-    "name": "Debug menu",
+    "name": "Developer debug menu",
```

```suggestion
    "name": "Debug menu",
```

making this change Kevin suggested and should be ready to merge once tests pass again.